generators/common/templates/dotnetcore/README.md.ejs

```diff
@@ -132,6 +132,13 @@ The `./src/<%= pascalizedBaseName %>/app/out` directory will contain your applic
 This will concatenate and minify the client CSS and JavaScript files. It will also modify `index.html` so it references these new files.
 <%_ } _%>
 
+## Code style / formatting
+
+To format the dotnet code, run
+
+    cd ./src/<%= pascalizedBaseName %>
```

Comment:
You can run dotnet format on root (on sln for format all solution)

Reply:
Thanks - fixed in the latest commit